README.md

```diff
@@ -8,7 +8,7 @@ Development Setup
 -----------------
 
   1. Install: `npm install`
-  2. Run the webserver: `gulp webserver`
+  2. Run the webserver: `./node_modules/.bin/gulp`
```

Comment:
Gulp recommends installing with `-g`, which would put it in PATH.  I guess this is fine though. (node hackers would know what you meant)